packages/node-red-s1seven-api/lib/utils/guards.js

```diff
@@ -0,0 +1,45 @@
+/// <reference path="typedefs.js" />
+
+/**
+ * @param {ApiConfig} apiConfig
+ * @returns {boolean}
+ */
+function isApiConfig(apiConfig) {
+  return (
+    apiConfig &&
+    typeof apiConfig === 'object' &&
+    typeof apiConfig.id === 'string' &&
+    // typeof apiConfig.clientId === 'string' &&
+    // typeof apiConfig.clientSecret === 'string' &&
+    typeof apiConfig.environment === 'string' &&
+    typeof apiConfig.apiVersion === 'number'
```

Comment:
I think `apiVersion` is stored as a string, currently the nodes crash with the following error:
```
[error] TypeError: apiConfig must be an object with apiVersion and environment
    at AsyncLocalStore.init (/Users/eamon/work/node-red-monorepo/packages/node-red-s1seven-api/lib/utils/async-local-storage.js:74:11)
    at hashCertificate._inputCallback (/Users/eamon/work/node-red-monorepo/packages/node-red-s1seven-api/lib/certificates/hash-certificate.js:39:25)
```
Updating `typeof apiConfig.apiVersion === 'string'` seems to fix the issue.